extensions/plugin-basic-ui/src/basicUIPlugin.tsx

```diff
@@ -57,6 +57,10 @@ export const basicUIPlugin: (
             compactMap({
               [theme.globalVars.backgroundColor]: options.backgroundColor,
               [theme.globalVars.dimBackgroundColor]: options.dimBackgroundColor,
+              [theme.globalVars.transitionDuration]:
+                stack.globalTransitionState === "loading"
+                  ? `${stack.transitionDuration}ms`
+                  : "0ms",
```

Comment:
```suggestion
                  : [theme.globalVars.transitionDuration],
```

아래 `theme.css.ts`에서 정의하신 `defaultVars` 쓰는건 혹시 별로일까요? 궁금해요!

Reply:
아하 그러면 차라리 해당 값을 undefined로 만들어버리는게 낫긴한데요. (그러면 루트의 0ms를 사용해요) 그러면 암묵적이여질것같네요. `idle` 상태에서는 transitionDuration이 `0ms` 라는게 명시적으로 드러나는걸 의도했어요.

Reply:
아하..! 저는 `theme.globalVars.transitionDuration` 자체가 `0ms`인걸 의도하시고 작성하신건줄 알았어요. 
다른 의도가 있으셨군요.

Reply:
참고:

```
theme.globalVars.transitionDuration === '0ms' // false
theme.globalVars.transitionDuration === 'var(--stackflow-transition-duration) // true
```